Source/Tests/LinuxSyscalls/FileManagement.cpp

```diff
@@ -138,13 +138,45 @@ void FileManager::LoadThunkDatabase(bool Global) {
           }
         }
         else if (strcmp(ItemName, "Overlay") == 0) {
+
+          auto AddWithReplacement = [DBObject](json_t const* Value) {
+            constexpr static std::array<const char*, 4> Prefixes = {
+              "/usr/lib",
+              "/usr/local/lib",
+              "/lib",
+              "/usr/lib/pressure-vessel/overrides/lib",
+            };
+
+            auto ReplacePrefix = [](const std::string &String, const char *Prefix, const char *NewPrefix) -> std::pair<bool, std::string> {
```

Comment:
Can avoid a heap allocation by typing `NonModifiedLibraryItem` as `const char*` and changing this parameter to `string_view` instead:
```suggestion
            auto ReplacePrefix = [](std::string_view &String, const char *Prefix, const char *NewPrefix) -> std::pair<bool, std::string> {
```

Reply:
Resolved.